### PR TITLE
Create PiiEntity

### LINKED ISSUE #9732: Change CategorizedEntity in Pii to PiiEntity

### PR DESCRIPTION
**Pull request by maririos**
Fixes: #9732

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]
LGTM

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesResult.cs [hunk 1]
```diff
@@ -9,28 +9,28 @@ namespace Azure.AI.TextAnalytics
...
+            PiiEntities = Array.Empty<PiiEntity>();
         }
 
         /// <summary>
-        /// Gets the collection of named entities containing Personally
+        /// Gets the collection of PII entities containing Personally
         /// Identifiable Information in the input document.
         /// </summary>
-        public IReadOnlyCollection<CategorizedEntity> CategorizedEntities { get; }
+        public IReadOnlyCollection<PiiEntity> PiiEntities { get; }
```
Comment (willmtemple) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesResult.cs]: I left a comment about this in the Teams channel, but are we set on renaming this key to `PiiEntities`? Seems fine to do so just want to make sure JS/TS are aligned as well.

**Review by willmtemple — APPROVED** [review]
all looks good to me, with one question

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesResult.cs (hunk 1 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesResult.cs]: will rename to `entities`

**Pull request merged**